ghcide/src/Development/IDE/Spans/Documentation.hs

```diff
@@ -65,9 +65,9 @@ lookupKind env mod =
     fmap (fromRight Nothing) . catchSrcErrors (hsc_dflags env) "span" . lookupName env mod
 
 getDocumentationTryGhc :: HscEnv -> Module -> Name -> IO SpanDoc
-getDocumentationTryGhc env mod n = head <$> getDocumentationsTryGhc env mod [n]
+getDocumentationTryGhc env mod n = fmap head ((fmap . fmap) snd $ getDocumentationsTryGhc env mod [n])
 
-getDocumentationsTryGhc :: HscEnv -> Module -> [Name] -> IO [SpanDoc]
```

Comment:
Provisioning the pairing through `unwrap` & it provisions & shows-up in the `getDocumentationsTryGhc` now.